kolibri/core/content/tasks.py

```diff
@@ -168,7 +169,7 @@ def remotechannelimport(channel_id, baseurl=None, peer_id=None):
 
 
 class RemoteChannelResourcesImportValidator(
-    RemoteImportMixin, ChannelResourcesImportValidator
+    RemoteImportMixin, ChannelResourcesValidator, ChannelValidator
```

Comment:
Could we fix the implementation of the `ChannelResourcesImportValidator` instead? Otherwise the same issue is going to happen for disk imports too.

Reply:
Done!